round2 of improvements (#11)
ofirgall/learn-nvim: chapters/00-why-should-i-learn.md

You can take a look at the [awesome nvim plugin list](https://github.com/rockerBOO/awesome-neovim). I recommend to "watch" to get updates about new plugins.

#### Be comfortable

> nvim is basically a framework for an editor, without plugins it can't do much but it gives the user the ability to configure it's editor in a way he is most comfortable with.

Reviewer: ```suggestion
nvim is basically a framework for an editor, it can't do much without plugins but it gives the user the ability to configure the editor in a way they're most comfortable with.
```

> Imaging sitting on a chair you built exactly to your size. The back rest covers your whole back, the head rest is exactly in the right height and has the perfect shape to support your head and neck. The arm rests are exactly in the height of your desk, the chair is in the perfect height for your legs. \

Reviewer: ```suggestion
Imagine sitting on a chair you built exactly to your size. The back rest covers all of your back, the head rest is exactly in the right height and has the perfect shape to support your head and neck. The arm rests match exactly the height of your desk, and the chair is at the perfect height for your legs. \
```
